systemvm/debian/opt/cloud/bin/setup/common.sh

```diff
@@ -322,7 +322,12 @@ setup_common() {
 
   if [ -n "$MGMTNET"  -a -n "$LOCAL_GW" ]
   then
-    ip route add $MGMTNET via $LOCAL_GW dev eth1
+    log_it "Received mgmt cidr : $MGMTNET"
+    cidrs=(${MGMTNET//,/ })
+    for cidr in ${cidrs[@]}
+    do
+      ip route add $cidr dev eth1
```

Comment:
@ravening I am wondering why there is no "via gateway" in route.

Reply:
> @ravening I am wondering why there is no "via gateway" in route.

@ravening any update?

Reply:
> @ravening I am wondering why there is no "via gateway" in route.

@weizhouapache do we need the "via gateway" ? it was not there in the original change

Reply:
@ravening the original line contained it (`ip route add $MGMTNET via $LOCAL_GW dev eth1`), can you explain why is removed?